cli/src/cli.rs

```diff
@@ -60,16 +61,16 @@ pub fn handle_source(source: Source) -> bool {
         segment: 0..0,
     });
 
-    let name = Name::new("<module>");
+    let name = Name::new("<repl>");
```

Comment:
```suggestion
    let name = Name::new("<module>");
```
This is also used while reading a file.